release-notes/6.0/api-diff/.Net/6.0.0_System.Diagnostics.CodeAnalysis.md

```diff
@@ -0,0 +1,16 @@
+# System.Diagnostics.CodeAnalysis
+
+``` diff
+ namespace System.Diagnostics.CodeAnalysis {
+     public enum DynamicallyAccessedMemberTypes {
++        Interfaces = 8192,
+     }
++    public sealed class RequiresAssemblyFilesAttribute : Attribute {
```

Comment:
What is this attribute for?

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.requiresassemblyfilesattribute?view=net-6.0 (which probably needs to be enhanced with some more explanation.

This https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/il3002 provides a pretty good explanation of what this attribute is for.